### DISCUSSION TITLE
PHD2 RPI and Player-One Camera (Neptune)

### DISCUSSION THREAD

**Discussion by JaySPA** [indilib/indi#1818] [General]
All..

Trying to get PHD2 to recognize a new Player-One Neptune-C II on Bullseye RPI4.  Ekos sees the camera, but PHD2 is not picking up the driver I don't think.  Appreciate any thoughts..

Jay S.

**Comment by knro** [comment]
What do you see in PHD2 exactly when you connect it to INDI server?

**Reply by JaySPA** [reply]
Thanks for the reply..  Didn't realize I have to start either INDIGO or INDI services to have it recognized in PHD2.  My issue is that I use SkyX on my RPI for most all activites, but prefer PHD2 for guiding. The X2 driver written for Player-One on Bullseye works fine as long as I'm in SkyX.  INDI and SkyX though do not tend to like each other as far as device control, so starting INDI up (or SkyX) tends to lock out hardware for the other.  Bottom line is I can get it working now on PHD2 for RPI, but then tends to interfere on SkyX side.  Trying to see if I can isolate them from each other when using non-ZWO hardware (cameras, filter wheel connections, etc.).

Jay S.
